.circleci/config.yml

```diff
@@ -5,15 +5,15 @@ workflows:
   build_and_test:
     jobs:
       - build
-      - "test-arangodb-3_4_8":
+      - "test-arangodb-3_4_9":
```

Comment:
Small point but next time we need to do this I think we could use `3_4`, `3_5, `3_6` for the step names and when defining `ARANGO_HOST` etc, that way we won't need to change them again when we want to start testing against the next minor update.

Reply:
i.e. drop the patch version specifier from the names, since I expect we'd only plan to test against the latest patch version for each minor version

Reply:
Good idea! I will update the changes, I think it's worth doing now.